qiskit/pulse/transforms/canonicalization.py

```diff
@@ -157,7 +157,9 @@ def _inline_schedule(schedule: Schedule) -> Schedule:
     """
     ret_schedule = Schedule(name=schedule.name,
                             metadata=schedule.metadata)
-    for t0, inst in schedule.instructions:
+    for t0, inst in schedule._children:
```

Comment:
Any way to avoid accessing a private variable?

Reply:
I don't think so, I would be fine with this becoming a public variable in a separate patch. The old pre-block builder had to use this as well.